pkg/build/android.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package build
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/google/syzkaller/pkg/osutil"
+)
+
+const (
+	kernelConfig = "common/build.config.gki_kasan.x86_64"
+	moduleConfig = "common-modules/virtual-device/build.config.virtual_device_kasan.x86_64"
+)
+
+type android struct{}
+
+func (a android) runBuild(kernelDir, buildConfig string) error {
+	cmd := osutil.Command("build/build.sh")
```

Comment:
Maybe also add here CC= and LD= for the sake of consistency with how it's done for other builders?

Syzkaller does not expect the `params.Compiler` to be silently ignored without setting details.CompilerID to something.
See:
https://github.com/google/syzkaller/blob/master/pkg/build/build.go#L79

It might lead to confusing bug reports.